pkgs/misc/drivers/epson-201202w/default.nix

```
@@ -0,0 +1,78 @@
+{ lib, stdenv, fetchurl, rpmextract, autoreconfHook, file, libjpeg, cups }:
+
+let
+  version = "1.0.0";
+  filterVersion = "1.0.0";
+in
+  stdenv.mkDerivation {
+
+    pname = "epson-201202w";
+    inherit version;
+
+    src = fetchurl {
```

would you rather let nix unpack these for you? (`fetchzip` supports tarballs too)
```suggestion
    src = fetchzip {
```